### PR TITLE
Spinnaker v0.3.0

### PR DESCRIPTION
**Pull request by viglesiasce**
1. Fixes using 127.0.0.1 for access to web ui
2. Fix install hooks when using GCS
3. Add flag for tracking releases by digest rather than just tags
4. Use Jenkins chart as a dependency with custom config map
5. Delete jobs using label rather than individually
6. Add Jenkins job to run pipeline directly from repo
7. Allow configurable image in Jenkins jobs

### CODE REVIEW

**Comment by askcarter** [comment]
I tested this and it worked for me.

Steps I used to test this out:

```
# Get the changes
$ git fetch origin pull/1338/head:test-chart
$ git checkout test-chart

# update the dependencies
$ helm dep up 

# Install spinnaker
$ helm install ../charts/stable/spinnaker --name cd -f values2.yaml --timeout 1500
```

From there I was able to set up port-forwarding and access the UI.

File: stable/spinnaker/requirements.yaml [hunk 1]
```diff
@@ -6,3 +6,6 @@ dependencies:
   version: 0.1.0
   repository: https://kubernetes-charts.storage.googleapis.com/
   condition: minio.enabled
+- name: jenkins
+  version: 0.6.3
```
Comment (prydonius) [inline stable/spinnaker/requirements.yaml]: want to update this to 0.8.0?

**Comment by viglesiasce** [comment]
Updated @prydonius PTAL

**Review by lachie83 — APPROVED** [review]
LGTM

**Comment by lachie83** [comment]
@viglesiasce please rebase then this should be GTG.

**Pull request merged**